src/client/eppo-client-with-bandits.spec.ts

```diff
@@ -429,5 +430,129 @@ describe('EppoClient Bandits E2E test', () => {
         expect(mockLogBanditAction.mock.calls[1][0].actionProbability).toBeCloseTo(0.256);
       });
     });
+
+    describe('Assignment logging deduplication', () => {
+      let mockEvaluateFlag: jest.SpyInstance;
+      let mockEvaluateBandit: jest.SpyInstance;
+      let variationToReturn: string;
+      let actionToReturn: string | null;
+
+      // Convenience method for repeatedly making the exact same assignment call
+      function requestClientBanditAction(): Omit<IAssignmentDetails<string>, 'evaluationDetails'> {
```

Comment:
I find `function` more descriptive than `const`, and since I don't need lexical `this` nor does this being hoisted matter, I'm thus opting to go for it.